src/components/organisms/ActionsPane/ActionsPane.tsx

```diff
@@ -382,92 +382,94 @@ const ActionsPane: React.FC = () => {
 
   return (
     <S.ActionsPaneMainContainer ref={actionsPaneRef} id="EditorPane" $height={height}>
-      <ActionsPaneHeader
-        actionsPaneWidth={actionsPaneWidth}
-        applySelection={applySelection}
-        selectedResource={selectedResource}
-      />
-
-      {selectedPreviewConfigurationId ? (
-        <PreviewConfigurationDetails />
-      ) : selectedImage ? (
-        <ImageDetails />
-      ) : selectedResourceId || selectedPath || selectedValuesFileId ? (
-        <S.Tabs
-          $height={height - DEFAULT_PANE_TITLE_HEIGHT}
-          defaultActiveKey="source"
-          activeKey={activeTabKey}
-          items={tabItems}
-          onChange={k => setActiveTabKey(k)}
-          tabBarExtraContent={
-            selectedResource && resourceKindHandler?.helpLink ? (
-              <>
-                <Tooltip mouseEnterDelay={TOOLTIP_DELAY} title={EditWithFormTooltip}>
+      <S.HeaderWrapper>
```

Comment:
Why was the HeaderWrapper needed since we already have the `ActionsPaneMainContainer`? Can we style that directly ?

Reply:
moved HeaderWrapper styles into ActionsPaneMainContainer and removed HeaderWrapper 👍